### PR TITLE
Install instruction for using Android BOM artifact

### PR DESCRIPTION
**Pull request by yangyansong-adbe**

### CODE REVIEW

File: src/pages/documentation/current-sdk-versions.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 **Migrate to latest Mobile SDKs for Android**<br/>If you are currently using the Mobile Core 1.x and the compatible libraries, please see the [Migrating to latest Mobile SDKs for Android](https://developer.adobe.com/client-sdks/previous-versions/documentation/migrate-to-android/) guide for next steps.<br/>The latest Mobile SDKs for Android support Google Android API 19 (KitKat) or later.
 
+<InlineAlert variant="info" slots="text"/>
+
+**Migrate to using BOM artifact for Android**<br/> The Adobe SDK BOM artifact enables managing all compatible versions of Adobe Experience Platform Android extensions by specifying a single BOM version. This is now the recommended way to manage Android SDKs.
```
Comment (calebk1) [inline src/pages/documentation/current-sdk-versions.md:11]: "Migrate to using the BOM artifact..."

**Review by calebk1 — COMMENTED** [review]
Just one change!

But one comment - what's the correct link to the manage-gradle-dependencies? I've seen it as both resources/manage-gradle-dependencies and not within resources.

We should confirm that it's the correct link!

**Comment by yangyansong-adbe** [comment]
@calebk1  https://github.com/AdobeDocs/aep-mobile-sdkdocs/blob/main/src/pages/documentation/manage-gradle-dependencies.md found it. It seems that we used the wrong links for each extension., I will fix it in this PR. Thank you for pointing it out.

**Review by calebk1 — APPROVED** [review]
LGTM!

File: src/pages/documentation/current-sdk-versions.md [hunk 2]
```diff
@@ -6,6 +6,10 @@
 
 **Migrate to latest Mobile SDKs for Android**<br/>If you are currently using the Mobile Core 1.x and the compatible libraries, please see the [Migrating to latest Mobile SDKs for Android](https://developer.adobe.com/client-sdks/previous-versions/documentation/migrate-to-android/) guide for next steps.<br/>The latest Mobile SDKs for Android support Google Android API 19 (KitKat) or later.
 
+<InlineAlert variant="info" slots="text"/>
+
+**Migrate to using the BOM artifact for Android**<br/> The Adobe SDK BOM artifact enables managing all compatible versions of Adobe Experience Platform Android extensions by specifying a single BOM version. This is now the recommended way to manage Android SDKs.
```
Comment (emdobrin) [inline src/pages/documentation/current-sdk-versions.md:11]: Migration is overloaded here, can we use:
Migrate to using the BOM ... -> Start using the BOM ...

File: src/pages/documentation/adobe-campaign-standard/tabs/index.md [hunk 3]
```diff
@@ -3,17 +3,18 @@
...
+implementation 'com.adobe.marketing.mobile:identity'
+implementation 'com.adobe.marketing.mobile:lifecycle'
+implementation 'com.adobe.marketing.mobile:signal'
+implementation 'com.adobe.marketing.mobile:userprofile'
 ```
 
 <InlineNestedAlert variant="warning" header="false" iconPosition="left">
 
-Using dynamic dependency versions is **not** recommended for production apps. Please read the [managing Gradle dependencies guide](../resources/manage-gradle-dependencies.md) for more information. 
+Using dynamic dependency versions is **not** recommended for production apps. Please read the [managing Gradle dependencies guide](../../manage-gradle-dependencies.md) for more information.
```
[the remark this answers is not in the document]
Comment (yangyansong-adbe) [reply src/pages/documentation/adobe-campaign-standard/tabs/index.md:17]: src/pages/documentation/adobe-campaign-standard/tabs/index.md
src/pages/documentation/manage-gradle-dependencies.md
`../resources/manage-gradle-dependencies.md` points to the right page. I think we can make changes in another PR if we want to use the `Alert`.

**Review by calebk1 — APPROVED** [review]
Looks good to me!

I left a comment behind about the future alert.

File: src/pages/documentation/adobe-campaign-standard/tabs/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (emdobrin) [reply src/pages/documentation/adobe-campaign-standard/tabs/index.md:17]: @yangyansong-adbe Campaign classic was using the path `../manage-gradle-dependencies.md` for src/pages/documentation/adobe-campaign-classic/tabs/index.md hence my suggestion. If you tested all these links in your local or staging environment you may disregard, if not please do a quick test with these changes and update where needed.

**Review by emdobrin — APPROVED** [review]

File: src/pages/documentation/adobe-campaign-standard/tabs/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (yangyansong-adbe) [reply src/pages/documentation/adobe-campaign-standard/tabs/index.md:17]: Oh, I see. I only tested it on the static page, I will test it on the local server.

[the remark this answers is not in the document]
Comment (yangyansong-adbe) [reply src/pages/documentation/adobe-campaign-standard/tabs/index.md]: @emdobrin  I have updated the PR to use `../manage-gradle-dependencies.md` in all occurrences.

**Pull request merged**